src/psram.h

```diff
@@ -0,0 +1,26 @@
+/**
+    @file sfe_psram.c
+
+  */
+
+#ifndef __psram_h__
+#define __psram_h__
+
+#include <stddef.h>
+
+void runtime_init_setup_psram(void);
+
+void psram_reinit_timing(void);
+void *__psram_malloc(size_t size);
```

Comment:
These are [invalid identifier names](https://en.cppreference.com/w/c/language/identifier).

> The following identifiers are reserved and may not be declared in a program (doing so invokes undefined behavior):
> 
> 1. The identifiers that are [keywords](https://en.cppreference.com/w/c/keyword) cannot be used for other purposes. In particular #define or #undef of an identifier that is identical to a keyword is not allowed.
> 2. All external identifiers that begin with an underscore.

Undefined behavior is to be avoided.

Rename these functions to have a prefix (e.g., `bp_` instead of `__`).